src/sas/qtgui/Perspectives/Fitting/ComplexConstraint.py

```diff
@@ -132,6 +154,12 @@ def setupParamWidgets(self):
             self.cmdAddAll.setEnabled(True)
         self.lblWarning.setText(txt)
 
+        # disable Aplly all if `All` option on lhs has been selected
+        if self.cbModel1.currentText() == "All":
+            self.cmdAddAll.setEnabled(False)
+        else:
+            self.cmdAddAll.setEnabled(True)
```

Comment:
Or maybe: `self.cmdAddAll.setEnabled(self.cbModel1.currentText() != "All")`

Reply:
The one liner is less readable and introduces no clear advantage, I think.